DEVELOPMENT.md

```diff
@@ -3,14 +3,28 @@
 * [Java 21](https://openjdk.org/install)
 * [Docker](https://docs.docker.com/engine/install)
 * [Node.js 18](https://nodejs.org/en/download) (or higher)
+* [Postgres](https://www.postgresql.org/download/)
```

Comment:
I don't think you need Postgres. It starts automatically in Docker.

Reply:
Yes, realized that later. I will update.